### PR TITLE
Slideshow effect to image slider of About page

### PR DESCRIPTION
**Pull request by pandeatul27**
## Related Issue
- Issue No. #19

## Changes you made : Gave Slideshow effect to the Image slider 
- 

## Check list :

- [x] I have followed code style of this project.
- [x] I have read the [**CONTRIBUTING**](https://github.com/Arun9739/Paryatana/blob/main/Contributing.md) document.
- [x] All the aspects mentioned in the issue are covered.
- [x] The website is properly working after my changes.
- [x] 🌟 ed the repo

### Type of change :

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -267,9 +267,9 @@ <h3 class="tm-testimonials-title">Testimonials</h3>
 			</div>
 		</div>		
 	</footer>
-	<script type="text/javascript" src="js/jquery-1.11.2.min.js"></script>      		<!-- jQuery -->
+	<script type="text/javascript" src="js/jquery-1.11.2.min.js"></script>      		jQuery
   	<script type="text/javascript" src="js/bootstrap.min.js"></script>					<!-- bootstrap js -->
-  	<script type="text/javascript" src="js/jquery.flexslider-min.js"></script>			<!-- flexslider js -->
+  	<script type="text/javascript" src="js/jquery.flexslider-min.js"></script>			flexslider js
```
Comment (Arun9739) [inline about.html:272]: Do the same for this also

**Review by Arun9739 — CHANGES_REQUESTED** [review]

**Comment by Arun9739** [comment]
Good work @pandeatul27 
But do the changes and resolve branch conflicts

**Comment by pandeatul27** [comment]
@Arun9739  I have done the changes, please review them.

**Review by Arun9739 — APPROVED** [review]

**Comment by pandeatul27** [comment]
@Arun9739  the branch conflicts have been resolved, sorry for the inconvenience

**Comment by Arun9739** [comment]
Good to go @pandeatul27 
LGTM 💯

**Pull request merged**